src/libsync/logger.cpp

```diff
@@ -67,10 +67,10 @@ Logger *Logger::instance()
 Logger::Logger( QObject* parent) : QObject(parent),
   _showTime(true), _logWindowActivated(false), _doFileFlush(false), _logExpire(0)
 {
-#ifndef NO_MSG_HANDLER
```

Comment:
:-1:  

I want to keep my own message pattern which i set in an environment variable.

(My message pattern has color code for warning which is very isefull.)

So please don't overwrite it when NO_MSG_HANDLER is defined.

Or maybe

#ifdef NO_MSG_HANDLER
if (!qEnvironmentVariableIsSet("QT_MESSAGE_PATTERN"))
#endif